mobile/src/pages/Login.tsx

```diff
@@ -11,6 +11,14 @@ export default function Login({ navigation }) {
   const [password, setPassword] = useState("");
   const [isPending, setIsPending] = useState(false);
 
+  //# region Styles
+
+  const textInputViewStyle = "mx-auto rounded-lg bg-slate-300 w-80 h-12 mb-5 items-start justify-center";
```

Comment:
This is one way to to do this, but I would suggest extracting this to a local component of its own. The `Text` and `TextInput` pairs here are the real duplication, not just the CSS classes. Lmk if you're confused about what I mean here.